posthog/queries/funnels/test/test_breakdowns_by_current_url.py

```diff
@@ -0,0 +1,146 @@
+                },
+            ],
+            "person3": [
+                # no trailing slash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/home", "$pathname": "/home"},
+                },
+                {
+                    "event": "terminate funnel",
+                    "timestamp": datetime(2020, 1, 2, 12, 2),
+                },
+            ],
+            "person4": [
+                # trailing slash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/home/", "$pathname": "/home/"},
+                },
+                # trailing hash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 2),
+                    "properties": {"$current_url": "https://example.com/home#", "$pathname": "/home#"},
+                },
+                # all the things
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 3),
+                    "properties": {"$current_url": "https://example.com/home/?#", "$pathname": "/home/?#"},
+                },
+                {
+                    "event": "terminate funnel",
+                    "timestamp": datetime(2020, 1, 2, 12, 4),
+                },
+            ],
+        }
+
+        journeys_for(journey, team=self.team, create_people=True)
+
+    def _run(self, extra: Dict = {}, events_extra: Dict = {}):
+        response = ClickhouseFunnel(
+            Filter(
+                data={
+                    "events": [
+                        {"id": "watched movie", "name": "watched movie", "type": "events", "order": 0, **events_extra},
+                        {
+                            "id": "terminate funnel",
+                            "name": "terminate funnel",
+                            "type": "events",
+                            "order": 1,
+                            **events_extra,
+                        },
+                    ],
+                    "funnel_viz_type": "steps",
+                    "insight": "FUNNELS",
+                    "date_from": "2020-01-02T00:00:00Z",
+                    "date_to": "2020-01-12T00:00:00Z",
+                    "breakdown_limit": 100,  # never have other
+                    **extra,
+                }
+            ),
+            self.team,
+        ).run()
+        return response
+
+    @snapshot_clickhouse_queries
+    def test_breakdown_by_pathname(self) -> None:
+        response = self._run({"breakdown": "$pathname", "breakdown_type": "event", "breakdown_normalize_url": True})
+
+        actual = []
+        for breakdown_value in response:
```

Comment:
what was the problem with 'other' ?

Reply:
Because I wasn't normalizing URLs in the inner query when we compared the candidate prop with the list of "not other" values it didn't match and was grouped into other

so if we had

/home
/home?
/home#

only /home would sit outside other